daemon/src/routes_taker.rs

```diff
@@ -211,3 +211,59 @@ pub fn index<'r>(_paths: PathBuf) -> impl Responder<'r, 'static> {
     let asset = Asset::get("index.html").ok_or(Status::NotFound)?;
     Ok::<(ContentType, Cow<[u8]>), Status>((ContentType::HTML, asset.data))
 }
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct WithdrawRequest {
+    pub address: bdk::bitcoin::Address,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
+    pub amount: Amount,
+    pub fee: f32,
+}
+
+#[rocket::post("/withdraw", data = "<withdraw_request>")]
+pub async fn post_withdraw_request(
+    withdraw_request: Json<WithdrawRequest>,
+    wallet: &State<Address<wallet::Actor>>,
+    network: &State<Network>,
+) -> Result<status::Accepted<String>, HttpApiProblem> {
+    tracing::info!(
+        "Withdrawing {:?} to {} with fee {}",
+        withdraw_request.amount,
+        withdraw_request.address,
+        withdraw_request.fee
+    );
+
+    let amount = if withdraw_request.amount == Amount::ZERO {
+        None
+    } else {
+        Some(withdraw_request.amount)
+    };
+
+    let txid = wallet
+        .send(wallet::Withdraw {
+            amount,
+            address: withdraw_request.address.clone(),
+            fee: Some(bdk::FeeRate::from_sat_per_vb(withdraw_request.fee)),
+        })
+        .await
+        .map_err(|e| {
+            HttpApiProblem::new(StatusCode::INTERNAL_SERVER_ERROR)
+                .title("Withdraw failed")
+                .detail(e.to_string())
+        })?
```

Comment:
The first `Result` only happens when the actor gets disconnected, I think we can write a better error message here (500 is still a good code).